server/routes/user.py

```diff
@@ -74,7 +78,37 @@ def index():
     return "Data Commons </br> <a href='/user/auth/login'><button>Login</button></a>"
 
 
+@bp.route('/upload', methods=['POST'])
+@login_is_required
+def upload():
+    # TODO: changeg SECRETE_PROJECT to APP_PROJECT
+    # Upload files to GCS
+    user_id = session["google_id"]
+    project = current_app.config['SECRET_PROJECT']
+    bucket_name = project + '-resources'
+    gcs_client = storage.Client(project=project)
+    bucket = gcs_client.get_bucket(bucket_name)
+    import_name = request.form.get('importName')
+    for f in request.files.getlist('files'):
+        blob_name = f'{user_id}/{import_name}/{f.filename}'
```

Comment:
Does this overwrite existing files? If a file of the same name already exists, should it be a warning? 

We could have a force flag in the request. If force flag is turned on, overwrite it. Otherwise uploading file of the same name returns 400.

Reply:
We would eventually have a user flow design and decide if user should overwrite an import or we save versioned import files. I like the idea of "forced" flag and also show it on UI to callout "overwrite". Will make this simple for now and enrich it in the next round